src/test/java/com/jcabi/github/mock/MkHookTest.java

```diff
@@ -51,6 +54,11 @@
      */
     private transient Hook hook;
 
+    /**
+     * XML encapsulated by the hook.
+     */
+    private transient XML xml;
```

Comment:
@llorllale I don't think we should use this xml, it breaks encapsulation; we shouldn't access this directly, see comments below.

Reply:
@paulodamaso I don't see this as an issue when breaking encapsulation in a test of the affected class